cogs/tickets.py

```diff
@@ -170,7 +170,9 @@ async def on_message(self, message: discord.Message):
             thread = self.ticket_cache[message.author.id]["remote"]
             thread = self.bot.get_channel(thread)
 
-            await self.thread_webhook.send(f"`{message.author}:` {message.content}", thread=thread)
+            await self.thread_webhook.send(
+                message.content, thread=thread, username=str(message.author), avatar_url=message.author.display_avatar
```

Comment:
If we use `.url` it will not be ignored if they have no pfp and will raise an error

Reply:
display_avatar if they have no pfp will work.

Reply:
as then it will have no issue.

Reply:
also f"{message.author}" just looks cleaner.

Reply:
it's most likely slower
also, `.url` will error if they have no pfp, not `display_avatar` (will return none)

Reply:
display_avatar can't return None.

Reply:
https://discordpy.readthedocs.io/en/latest/api.html#discord.User.display_avatar